recipes/sfml/all/patches/2.6.0-0006-disable-warning-flags.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/cmake/Macros.cmake b/cmake/Macros.cmake
+index e0118eb..85a7714 100644
+--- a/cmake/Macros.cmake
++++ b/cmake/Macros.cmake
+@@ -66,8 +66,6 @@ macro(sfml_add_library target)
+         add_library(${target} ${THIS_SOURCES})
+     endif()
+ 
+-    set_file_warnings(${THIS_SOURCES})
```

Comment:
A bit unfortunate that setting `WARNINGS_AS_ERRORS` to `OFF` was not sufficient due to newer warnings being referenced than gcc 5 supports.
I've opened an issue with upstream sfml: https://github.com/SFML/SFML/issues/2594

Chances are gcc 5 is no longer supported - if this is the case, we may chose to stop supporting it for this recipe as well.

Reply:
Thanks for quick action!

Reply:
> Chances are gcc 5 is no longer supported

I'm simply a matter of nobody using GCC 5 with SFML 2 anymore. If you removed those warnings I bet it would work. It's not as if the maintainers made an intentional decision to stop supporting GCC 5.